osu.sln.DotSettings

```diff
@@ -115,7 +115,7 @@
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=ReplaceWithSingleCallToAny/@EntryIndexedValue">WARNING</s:String>
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=ReplaceWithSingleCallToCount/@EntryIndexedValue">WARNING</s:String>
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=ReplaceWithSingleCallToFirst/@EntryIndexedValue">WARNING</s:String>
-	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=ReplaceWithSingleCallToFirstOrDefault/@EntryIndexedValue">WARNING</s:String>
+	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=ReplaceWithSingleCallToFirstOrDefault/@EntryIndexedValue">HINT</s:String>
```

Comment:
This can actually be an issue on anything which implements Linq; `FirstOrDefault` is not always correct and may come up in the future.

I think it's more important of us to be aware of this in code reviews than rely on an incorrect resharper rule (in this case, resharper doesn't check the signature of the `FirstOrDefault` method, and thus fails to see the lack of local implementation of `FirstOrDefault(predicate)` for `sqlite-net`